<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PÃ¡gina inicial</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 15, 15);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49,54,65);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49,54,65);
color: rgb(94,141,255);
font: 87 20pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Assistente Escolar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49,54,65);
color: rgb(94,141,222);
font: 87 12pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>8.0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_aulas">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(0,0,245);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>AULAS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_agenda">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(0,0,245);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>AGENDA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_sair">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(0,0,245);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_config">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0,0,245);
background-color: rgb(39,44,55);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>CONFIG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_site">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39,44,55);
color: rgb(0,0,245);
font: 87 10pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>ABRIR SITE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
